tests/many_to_one/models.py

```diff
@@ -98,3 +98,18 @@ class School(models.Model):
 
 class Student(models.Model):
     school = models.ForeignKey(School)
+
+
+class UnsavedForeignKey(models.ForeignKey):
```

Comment:
Since it doesn't seem that the database tables are required, could we inline these classes in the test?